processor/src/operations/decorators/debug_tests.rs

```diff
@@ -0,0 +1,171 @@
+#[cfg(test)]
+mod tests {
```

Comment:
Similar to the other comment: since `debug_tests.rs` is already a test module, there is no need to have internal `test` module as well.